lib/Jsrt/Jsrt.cpp

```diff
@@ -3491,36 +3468,44 @@ CHAKRA_API JsTTDStartTimeTravelRecording()
 #if !ENABLE_TTD
     return JsErrorCategoryUsage;
 #else
-    return ContextAPIWrapper<true>([&](Js::ScriptContext *scriptContext) -> JsErrorCode {
-        ThreadContext* threadContext = scriptContext->GetThreadContext();
-        if (threadContext->TTDLog == nullptr)
-        {
-            AssertMsg(false, "Need to create in TTD mode.");
-            return JsErrorCategoryUsage;
-        }
+    JsrtContext *currentContext = JsrtContext::GetCurrent();
+    JsErrorCode cCheck = CheckContext(currentContext, true);
+    if(cCheck != JsNoError)
+    {
+        AssertMsg(false, "This shouldn't happen!!!");
```

Comment:
>AssertMsg(false, "This shouldn't happen!!!"); [](start = 8, length = 45)

nit: Seems like not a very useful assert given that this is external API and can be called when this is supposed to fail like when context is not set.